doc/src/site/sphinx/tutorials/sw_xgboost.rst

```diff
@@ -36,6 +36,11 @@ Train the model. You can configure all the available XGBoost arguments using pro
     val estimator = new H2OXGBoost().setLabelCol("AGE")
     val model = estimator.fit(sparkFrame)
 
+You can also get H2O's binary model after calling *fit* method on the estimator as:
```

Comment:
nit: the \*fit* method? on->of?

Reply:
Thx! What do you think about this rewording? `You can also get H2O's binary model by calling *fit* method available on the estimator as:` ?

Reply:
I would also add 'the' before *fit*, otherwise LGTM :-)